guide/16-geoenrichment-guides/part1_introduction_to_geoenrichment.ipynb

```diff
@@ -0,0 +1,972 @@
+{
```

Comment:
<p>"To <s>access </s>(enable) GeoEnrichment, an ArcGIS Online subscription is needed or (your) ArcGIS Enterprise needs to be configured with GeoEnrichment utility service. (Please keep in mind that) GeoEnrichment operations consume credits."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
this is fixed.